### DISCUSSION TITLE
What does everyone use for notifications?

### DISCUSSION THREAD

**Discussion by Retalak** [sct/overseerr#3588] [Q&A]
What are you guys using for notifications? I have not been using them as of late because none of them work for me or fit my needs.

I want something that can message each user individually for their requests, which leaves email, Pushbullet, and Pushover at the moment.

Email is the obvious and perfect choice at a first glance. However, every free email provider I have tried that works will ban/disable your account if you use it as a service account in this manner.

Pushbullet does not support IOS, which many of my users have so that is out of the question.

Pushover would require each individual user to spend $5, which is out of the question.

So that basically leaves the non user-specific notification options like Discord or Slack. But I don't want every user to see notifications for every other user.

**Comment by jarzebowsky** [comment]
Use web push? That should resolve all of your issues. It works right now also on iOS so no problem from that part :)

**Reply by Retalak** [reply]
I wound up using Resend along with a domain to send emails. Someone on the discord recommended it to me and it works fantastically.

**Reply by ZeroFux78** [reply]
How did you get the iOS notifications working? I have web push enabled and I see the test notification on Firefox, but when I have Overseerr in Safari on my iPhone, nothing comes through

**Reply by jarzebowsky** [reply]
@ZeroFux78 
1. Enter your overseer website address in safari on iOS
2. Click share button
3. Select `Add to Home Screen`
4. Go to `Settings` > `Apps` > `Overseer` and make sure you allow notifications
5. ??
6. Profit 😁
